modules/customizing-the-web-console-URL.adoc

```diff
@@ -3,67 +3,60 @@
 // * web_console/customizing-the-web-console.adoc
 
 [id="customizing-the-web-console-url_{context}"]
-= Customizing the web console URL
+= Customizing console routes
 
-You can update the web console URL, `consoleURL`, to a custom value.
+For `console` and `downloads` routes, custom routes functionality is now implemented to use the new `ingress` config route configuration API. The Console Operator already contained custom route customization, but for the `console` route only. Now, if the `console` custom route is set up in both the `ingress` config and `console-operator` config, then the new `ingress` config custom route configuration takes precedent, because the route configuration via `console-operator` config is being deprecated.
+
+[id="customizing-the-console-route_{context}"]
+== Customizing the console route
+
+.Prerequisites
+
+* Log in to the cluster as a user with administrative privileges.
 
 .Procedure
 
-. Modify the cluster instance created by default during installation in the `consoles.operator.openshift.io` custom resource:
-+
-[source,terminal]
-----
-$ oc patch consoles.operator.openshift.io cluster --patch '{"spec":{"route":{"hostname":"console.example.com"}}}' --type=merge
-----
+* Set the custom hostname and, if needed, set the TLS on the ingress config's spec:
```

Comment:
"the TLS" seems awkward to me.  I've read "the TLS <something>" in many places or just something like "...set TLS on the..."

Can you change to "the config spec for the ingress" too?